deploy/kubernetes/deploy-driver.sh

```diff
@@ -16,6 +16,29 @@ set -o errexit
 readonly PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
 readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
 
+function check_service_account()
+{
+	# Using bash magic to parse JSON for IAM
+	IAM_NAME=$(grep -Po '"client_email": *\K"[^"]*"' ${GCE_PD_SA_DIR}/cloud-sa.json | tr -d '"')
+	PROJECT=$(grep -Po '.*@\K[^.]+'<<<$IAM_NAME)
+	GOTTEN_BIND_ROLES=$(gcloud projects get-iam-policy $PROJECT --flatten="bindings[].members" --format='table(bindings.role)' --filter="bindings.members:$IAM_NAME")
+	readonly BIND_ROLES="roles/compute.storageAdmin roles/iam.serviceAccountUser projects/${PROJECT}/roles/gcp_compute_persistent_disk_csi_driver_custom_role"
+	MISSING_ROLES=false
+	for role in $BIND_ROLES 
```

Comment:
nit: all variables in `"${var}"` form for consistency. I think the quotes prevent/fix some sort of multi-part variable issues